package.json

```diff
@@ -84,6 +84,7 @@
     "@types/react": "^17"
   },
   "dependencies": {
+    "@react-theming/storybook-addon": "^1.1.7",
```

Comment:
Why do we need this `dependency`?

Reply:
It's needed to be able to apply the theming into the preview.js file. It wasn't working without it.
https://storybook.js.org/addons/@react-theming/storybook-addon